nkms/config/interface.py

```diff
@@ -0,0 +1,11 @@
+"""
+Public facing client interface
+"""
+
+from nkms.config.keys import KMSKeyring
+
+
+def _bootstrap_config():
+    """Do not actually use this."""
+    passphrase = input("Enter passphrase >> ")
```

Comment:
Use `getpass` for this instead of `input`.

Reply:
:+1: